pandas/core/internals/blocks.py

```diff
@@ -2544,6 +2550,40 @@ def setitem(self, indexer, value):
                               klass=ObjectBlock,)
             return newb.setitem(indexer, value)
 
+    def quantile(self, qs, interpolation='linear', axis=0, axes=None):
+        # TODO: Add quantile as a reduction method.
+        # We can't just use Block.quantile, as that converts the DTA
+        # to an ndarray[object] via get_values.
+        # This method
+        # 1. Convert DatetimeTZBlock -> DatetimeBlock
+        # 2. Perform the op via Block.quantile
+        # 3. Converts back to tz-aware
+        # Alternatively, we could special case the call to `get_values`
```

Comment:
Thoughts on this alternative? I didn't think of it until I finished writing this.

Basically just a

```
if is_datetimetz_dtype(self.dtype):
    values = self.get_values(dtype=_NS_DTYPE)
else:
    values = self.get_values()
```

in Block.quantile, I hope.

Reply:
your suggestion here seems good?